scanners/flutter/flutter.go

```diff
@@ -225,15 +230,20 @@ projects:
 		scanner.projects = append(scanner.projects, proj)
 	}
 
-	if len(scanner.projects) == 0 {
-		return false, nil
+	for _, project := range scanner.projects {
+		proj := flutterproject.New(project.path, flutterproject.NewFileOpener())
+		sdkVersions, err := proj.FlutterAndDartSDKVersions()
+		if err == nil {
+			scanner.tracker.LogSDKVersions(sdkVersions)
+		}
 	}
+	scanner.tracker.Wait()
```

Comment:
Is it necessary to wait here?

Why not make logging analytics a fire and forget operation or move the call to `Wait` to just before exiting the program?

Reply:
I just wanted to make sure that we log the data before the scanner gets freed up.